kickstart-london.tex

```diff
@@ -0,0 +1,167 @@
+% Footnotes used inline in the text.
+\newcommand{\manualhandlingfootnote}{\footnote{\url{http://www.hse.gov.uk/pubns/indg143.pdf}}}
+\newcommand{\chargingfootnote}{\footnote{\url{https://studentrobotics.org/docs/kit/batteries/}}}
+\newcommand{\estatesfacilitiesfootnote}{\footnote{Estates and Facilities: \url{http://www.southampton.ac.uk/estates/}}}
+\newcommand{\hscfootnote}{\footnote{Health and Safety Coordinator; for the duration of this event, this role will be filled by Andrew Barrett-Sprot (\email{abarrett-sprot@studentrobotics.org}).}}
+
+
+\newcommand{\groupname}{Student Robotics}
+\newcommand{\assessorname}{Andrew Barrett-Sprot}
+\newcommand{\assessoremail}{abarrett-sprot@studentrobotics.org}
+\newcommand{\assessmentdate}{October 20, 2018}
+
+
+\newcommand{\activityname}{Student Robotics 2019 - Kickstart}
+\newcommand{\activitydate}{November 10, 2018}
+\newcommand{\activitytime}{All day}
+\newcommand{\activitylocation}{
+    Thread, 1 Alie St, London E1 8DE, UK
+}
+\newcommand{\activitysummary}{
+    This is the opening event to a five-month robotics competition in  which
+    teams of students from schools will compete to design, build and test
+    autonomous robots. The robots must perform the simple task of locating and
+    moving marked boxes around an arena.
+
+    The students are aged 16--18 and will be supervised by at least one teacher
+    from their school at all times.
+
+    The programme is run by the charity Student Robotics, who
+    will be considered event staff at this and future events.
+
+    The schedule of the event is:
```

Comment:
I'm not sure how the schedule relates to this being a risk-assessment?
If we are keeping it, maybe it should reflect the proposed schedule -- that the talks are about 45 minutes and then the teams are into prep/μgames.

Reply:
Or perhaps merely summarise the overview of the day -- that it contains some talks and then a long workshop session?
I don't know how much detail is expected for this sort of thing though, so if the detail is needed then obviously lets just update it to be correct.

Reply:
I suspect having a schedule is overkill, we should really just highlight the fact that there's a workshop session